primitives/Cargo.toml

```diff
@@ -20,6 +20,7 @@ default = ["std"]
 std = [
 	"codec/std",
 	"scale-info/std",
+	"sp-consensus-aura/std",
```

Comment:
why did you need to add this? You aren't using it anywhere

Reply:
I originally added the decimal conversion to the `CustomMetadata` struct. We were missing this dependency to be able to run the `primitives` module in isolation. To verify:

```
git checkout master; cd primitives; cargo check
```